packages/framework/src/Model/ContactForm/ContactFormFacade.php

```diff
@@ -31,13 +31,14 @@ public function __construct(
      */
     public function sendMail(ContactFormData $contactFormData)
     {
+        $mainAdminMail = $this->mailSettingFacade->getMainAdminMail($this->domain->getId());
         $messageData = new MessageData(
-            $this->mailSettingFacade->getMainAdminMail($this->domain->getId()),
+            $mainAdminMail,
             null,
             $this->getMailBody($contactFormData),
             t('Contact form'),
-            $this->mailSettingFacade->getMainAdminMail($this->domain->getId()),
-            $contactFormData->name,
+            $mainAdminMail,
+            $mainAdminMail,
```

Comment:
`$this->mailSettingFacade->getMainAdminMailName($this->domain->getId())` would be more suitable here - there is "Email name" setting in the administration that should be used :wink: 

![image](https://github.com/shopsys/shopsys/assets/10401898/e3e1457f-840d-4063-bd57-95f752c86055)